Added a new media in img for the translated diagram in English

allezalex opened the pull request:
## 📰 Short Description 
Added a new media in img for the translated diagram in English, referenced it in Configuration and made it the default diagram for all translations except FR.
+ Upscaled the FR diagram to be in better quality!

## 📸 Screenshots 
<img width="707" alt="Screenshot 2025-03-16 at 17 26 41" src="https://github.com/user-attachments/assets/05b1cd2e-0e99-411b-bdaa-9853eda37d7d" />
<img width="468" alt="Screenshot 2025-03-16 at 17 26 13" src="https://github.com/user-attachments/assets/de05cc72-6687-417f-bb04-3f22df062c5c" />

allezalex commented:
Disregard the second screenshot, realized I misrefered the file, corrected it in latest push!

frgfm commented on img/.DS_Store:
We need .DS_Store in the gitignore. Can you first remove all of those, commit, then add these to the gitignore?

allezalex replied:
Added and deleted

frgfm commented on img/solution.png:
Open question, the image is 3.5 larger, and thus longer to load. Should we keep a smaller size?

allezalex replied:
Reduced image to be under 500kb, which should be relatively quick to load !

frgfm commented on .DS_Store:
Let's add those in gitignore
```
*.DS_Store
```

allezalex replied:
Done

allezalex commented:
Hey good catch, deleted file and added it to our gitignore conditions !
Regarding the image size, since this is our key image to display I think we can allow ourselves to have it slightly larger than average, but reduced to under 500kb to ensure quicker loading time !

frgfm reviewed (approved):
Thanks Alex!
